chia/cmds/peer.py

```diff
@@ -0,0 +1,157 @@
+from typing import Optional, Dict
+
+import click
+
+from chia.util.network import is_trusted_inner
+
+
+async def print_connections(client, time, NodeType, trusted_peers: Dict):
```

Comment:
If you need the `time` module, go ahead and import it.

```suggestion
async def print_connections(client, NodeType, trusted_peers: Dict) -> None:
```